catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -144,7 +168,7 @@
  * services.
  */
 public abstract class AbstractCswSource extends MaskableImpl
-        implements FederatedSource, ConnectedSource, ConfiguredService {
+        implements CatalogStore, FederatedSource, ConnectedSource, ConfiguredService {
```

Comment:
We might consider renaming AbstractCswSource to AbstractCswStore but the thought is that the core CRUD operations should all be together.